manifests/ArturKovacs/emulsion/4.0.yaml

```
@@ -0,0 +1,14 @@
+Id: ArturKovacs.emulsion
+Version: 4.0
+Name: emulsion
+Publisher: ArturKovacs
+License: MIT
+LicenseUrl: https://raw.githubusercontent.com/ArturKovacs/emulsion/master/LICENSE.txt
+AppMoniker: emulsion
+Tags: image
```

```suggestion
Id: ArturKovacs.Emulsion
Version: 4.0
Name: EMULSION
Publisher: ArturKovacs
License: MIT
LicenseUrl: https://raw.githubusercontent.com/ArturKovacs/emulsion/master/LICENSE.txt
AppMoniker: emulsion
Tags: image, viewer
```

Caps and an extra tag. I put the name in full caps because that's how the website stylizes it.